### PR TITLE
Arr 102

### LINKED ISSUE #3: ARR-102
We are going to use [Jikan Manga Anime API](https://jikan.docs.apiary.io/#)  as discussed in ARR-101.
I want a simple PoC(Proof of Concept) for the above API.
## Acceptance Criteria:
Create the PR named ARR-102 for above.

## Approach
 - js file with get API calls.

### PR DESCRIPTION
**Pull request by jerin-ignatious**
Implemented PoC(Proof of Concept) for the Jikan Manga API calls.

### CODE REVIEW

**Comment by ShreyasSubhedar** [comment]
Great Work on API @jerin-ignatious, can you make global variables for 200 & 300 with their status ( ex 200 should be STATUS_SUCCESS).
overall it looks perfect.

File: src/App.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ function App() {
       <h3>{anime.length} episodes airing today</h3>
       <List anime={anime}/>
       <button onClick={() => setAnime([])}>
-        clear al
+        clear all
       </button>
     </section>
```
Comment (ShreyasSubhedar) [inline src/App.js]: make global variables

Comment (jerin-ignatious) [reply src/App.js]: @ShreyasSubhedar the said changes have been made.

**Review by ShreyasSubhedar — APPROVED** [review]
Nicely Done ✅

**Pull request merged**